src/components/live-image.tsx

```diff
@@ -155,6 +157,8 @@ type LiveImageShape = TLBaseShape<"live-image", { w: number; h: number }>;
 export class LiveImageShapeUtil extends ShapeUtil<LiveImageShape> {
   static override type = "live-image" as const;
 
+  override canResize = () => false;
```

Comment:
Disabled resizing on the custom shape as it doesn't have a resize function defined yet.